files/root/grafana_tools/white_label/README.md

```diff
@@ -0,0 +1,13 @@
+# Grafana White Label
+
+Script/Example Image files for giving Grafana a "White Label" makeover for your organization
+
+This is configured to work with Grafana v8
+
+A few notes:
+* Images provides are examples, replace with your own
```

Comment:
Minor: typo, should be `Images provided`

Reply:
Again, these files come from another git repo. I don't want to start editing or recreating the files.